src/coreclr/vm/amd64/theprestubamd64.S

```diff
@@ -6,17 +6,20 @@
 #include "asmconstants.h"
 
 NESTED_ENTRY ThePreStub, _TEXT, NoHandler
-        PROLOG_WITH_TRANSITION_BLOCK 0, 0, 0, 0, 0
+        PROLOG_WITH_TRANSITION_BLOCK 8, 0, 0, 0, 0
+        mov             [rsp + __PWTB_FloatArgumentRegisters - 8], rax
```

Comment:
```suggestion
        mov             [rsp + __PWTB_FloatArgumentRegisters - 8], rax // Return buffer in Swift calling convention
```

Reply:
We use `SIZEOF_MAX_OUTGOING_ARGUMENT_HOMES` to refer to the extra locals in the other place where they are used:

Reply:
Ok, I looked into a wrong file. I would just use [rsp] then.